lib/utils/data/app_list.dart

```diff
@@ -53,9 +53,11 @@ List<Application> applications = [
       iconName: "calculator",
       category: ApplicationCategory.office,
       description: "Solve mathematic calculations"),
-  const Application(
+  // ignore: prefer_const_constructors
```

Comment:
what, make it const, if it tells you stuff its because the entire variable should be const

if every item in a list is const then the entire list is const